network_plugin/network.py

```diff
@@ -120,7 +121,15 @@ def creation_validation(vca_client, **kwargs):
     static_ip = _split_adresses(get_mandatory(net_prop, 'static_range'))
     check_ip(static_ip.start)
     check_ip(static_ip.end)
-    check_ip(get_mandatory(net_prop, "dns"))
+    dns_list = get_mandatory(net_prop, "dns")
+    if dns_list:
+        check_ip(dns_list[0])
```

Comment:
Looks like you can just iterate over dns_list:
for ip in dns_list:
    check_ip(ip)